src/chart.zoom.js

```diff
@@ -201,8 +201,13 @@ function panIndexScale(scale, delta, panOptions) {
 
 function panTimeScale(scale, delta, panOptions) {
 	var options = scale.options;
-	options.time.min = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
-	options.time.max = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	
+	var newMax = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	var newMin = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
+	if (newMax != options.time.max && newMin != options.time.min) {
```

Comment:
Ah, ok.. You're talking about rounding errors when doing floating-point computations, where it's not a good idea to just check for equality, right?

This applies to `panNumericScale` (which I don't use and haven't tested O:-) ). 
I will try to refactor this one and add an epsilon factor.

But in this case `panTimeScale` is doing it's calculations using momentjs-dates. Checking for equality on them should be ok, shouldn't it?